x/authority/keeper/policies.go

```diff
@@ -0,0 +1,39 @@
+package keeper
+
+import (
+	"github.com/cosmos/cosmos-sdk/store/prefix"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/zeta-chain/zetacore/x/authority/types"
+)
+
+// SetPolicies sets the policies to the store
+func (k Keeper) SetPolicies(ctx sdk.Context, policies types.Policies) {
+	store := prefix.NewStore(ctx.KVStore(k.storeKey), types.KeyPrefix(types.PoliciesKey))
+	b := k.cdc.MustMarshal(&policies)
+	store.Set([]byte{0}, b)
```

Comment:
Why not just set the policy using the policy key as an index ?

Reply:
It's easier to maintain as a self-contained structure and easier to represent the state transition.
I think there is a need for a new key if the structure would grow over time